### DISCUSSION TITLE
BUG? variable has ambigue value?

### DISCUSSION THREAD

**Discussion by carrapaz** [fonsp/Pluto.jl#2784] [Questions about using Pluto] [answered]
so i'm starting out with cuda if i run a cell i get the current value for a but when i run the cell with the function a doesn't update in the others is this normal?
![image](https://github.com/fonsp/Pluto.jl/assets/52786458/8537c69b-c0e3-480b-8a01-a121d4795342)

**Comment by disberd** [comment, marked as answer]
Yes this is normal, what you are doing in the cell which calls `f` is _mutating_ the value of a, and Pluto [does not track mutations](https://github.com/fonsp/Pluto.jl/issues/564) for cell re-run.

**Reply by carrapaz** [reply]
ok thanks for the clarification :)
